### ISSUE TITLE
Button for menu does not have accessible name

### ISSUE THREAD

**Issue by airbr** [crickets-and-comb/crickets-and-comb.github.io#24]
**Describe the bug**
The Hamburger menu does not have text inside of it so it currently is announced as "button" by screen readers. 

Adding an `aria-label` is one way to improve this. This single attribute will give the button an accessible name. 

**To Reproduce**
1. Go to index.html
2. Resize viewport to tablet or mobile view
3. Run Lighthouse test or Axe Dev Tools

**Expected behavior**
Pass the audit, button have an accessible name

**Screenshots**

<img width="794" alt="Image" src="https://github.com/user-attachments/assets/7da32e98-6b35-41e1-b274-097eadcb0571" />

**Desktop (please complete the following information):**
 - Firefox
 - Mac OS
 - Desktop

**Additional context**
Enjoying the process you have! Ill make a PR after Im enabled or its ready to be submitted.

**Comment by KalebCoberly** [comment]
@airbr , thanks for doing this! I've added you to the list of allowed users for the other PR, so whenever you submit this one, it should stay open.
